scalafmt-interfaces/src/main/java/org/scalafmt/interfaces/Scalafmt.java

```diff
@@ -74,6 +73,11 @@
      */
     Scalafmt withReporter(ScalafmtReporter reporter);
 
+    /**
+     * Use this repositories to resolve dependencies.
+     */
+    Scalafmt withRepositories(String ... repositories);
```

Comment:
Let's make it clear that these must be Maven repositories
```suggestion
    Scalafmt withMavenRepositories(String ... repositories);
```

Reply:
no problem, I will amend the commit if you don't mind